docs/env.md

```diff
@@ -379,9 +388,45 @@ Whether SSL certificate validation in requests made by the AWS lambdas should be
 
 #### TF_VAR_migrations
 
-A comma separated list of migrations to execute by the migration lambda without the extension (`.js`). List of migrations are available in `src/aws/lambda-migrate/src/migrations`.
+A list of migrations to execute by the migration lambda without the extension (`.js`). List of migrations are available in `src/aws/lambda-migrate/src/migrations`.
 To execute the migration `0001_encode_timed_text_tracks.js` set this variable with `0001_encode_timed_text_tracks`
 
-- Type: string
+Example: ["0001_encode_timed_text_tracks"]
+
+- Type: string list
 - Required: No
+- Default: []
+
+#### TF_VAR_lambda_image_name
+
+The [ECR](https://aws.amazon.com/ecr/) image name you want to use. All lambda are built in one docker image and hosted on this repo
```

Comment:
```suggestion
The [ECR](https://aws.amazon.com/ecr/) image name you want to use. All lambdas are built in one docker image and hosted in this registry.
```

Reply:
No we create a repository: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecr_repository

Reply:
oh ok. Then write "repository" in full?

Reply:
Yes you are right